master_duff.py

```diff
@@ -52,7 +62,6 @@
     "**henry**": "henry",
     "*henry*": "henry",
     "__henry__": "henry",
-    "henry_0000": "henry",
```

Comment:
Not a big deal, but the fact that these are being removed means the version of the file you are using is slightly behind the latest master branch in my repo (I added these updated nicknames for Henry and Norah right after we chatted back in Dec)

See if you can pull the latest master into your repo, then these changes should disappear from the pull request